docs/proposals/proposal-template.md

```diff
@@ -0,0 +1,59 @@
+# Title
```

Comment:
We also need to specify the format of the title. Since we create an issue to discuss each proposal, let's use the issue number as the prefix. 

xxxx-<proposal-filename>.md

Here xxxx is the four-digit issue number.